<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Biblioteca</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="usuariosCadastradosBTN">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="publicacoesBTN">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="emprestimosBTN">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="paginas">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="inicialPG"/>
      <widget class="QWidget" name="usuariosPG">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Pagina de Usuários</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="buscarPorLB">
              <property name="text">
               <string>Buscar por</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="tipoDeBuscaUsuarioCB">
              <item>
               <property name="text">
                <string>Matricicula</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Nome</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>E-mail</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="pesquisaUsuarioLE"/>
            </item>
            <item>
             <widget class="QPushButton" name="buscarBTN">
              <property name="text">
               <string>Buscar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="usuarioTable"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="publicacoesPG">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Pagina de publiações</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Bucar por</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="tipoDeBuscaPublicacaoCB">
            <item>
             <property name="text">
              <string>Título</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Editora</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gênero</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="publicacaoLivroCheck">
            <property name="text">
             <string>Livro</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="publicacaoRevistaCheck">
            <property name="text">
             <string>Revista</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="publicacaoPeriodicosCheck">
            <property name="text">
             <string>Periódicos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="publicacaoArtigoCheck">
            <property name="text">
             <string>Artigo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="pesquisaPublicacaoLE"/>
          </item>
          <item>
           <widget class="QPushButton" name="buscarPublicacoesBTN">
            <property name="text">
             <string>Buscar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="publicacoesTable"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="emprestimoPG">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Pagina Emprestimos</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Pesquisar por</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="tipoDeBuscaEmprestimoCB">
            <item>
             <property name="text">
              <string>Nome do usuário</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Id do exemplar</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Definir data</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="definirDataEmprestimoCheck">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dataEmprestimo">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
            <property name="date">
             <date>
              <year>2000</year>
              <month>1</month>
              <day>6</day>
             </date>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLineEdit" name="pesquisaEmprestimoLE"/>
          </item>
          <item>
           <widget class="QPushButton" name="buscarEmprestimosBTN">
            <property name="text">
             <string>Buscar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="emprestimoBTN"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCadastro">
    <property name="title">
     <string>Cadastro</string>
    </property>
    <widget class="QMenu" name="menuUsuario">
     <property name="title">
      <string>Usuário</string>
     </property>
     <addaction name="actionAlunoCad"/>
     <addaction name="actionProfessorCad"/>
    </widget>
    <addaction name="menuUsuario"/>
    <addaction name="actionFuncionario"/>
    <addaction name="actionPublicacao"/>
   </widget>
   <widget class="QMenu" name="menuEmprestimo">
    <property name="title">
     <string>Emprestimo</string>
    </property>
    <addaction name="actionAlunoEmp"/>
    <addaction name="actionProfessorEmp"/>
   </widget>
   <widget class="QMenu" name="menuDevolucao">
    <property name="title">
     <string>Devolução</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
   </widget>
   <addaction name="menuCadastro"/>
   <addaction name="menuEmprestimo"/>
   <addaction name="menuDevolucao"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFuncionario">
   <property name="text">
    <string>Funcionário</string>
   </property>
  </action>
  <action name="actionPublicacao">
   <property name="text">
    <string>Publicação</string>
   </property>
  </action>
  <action name="actionAlunoCad">
   <property name="text">
    <string>Aluno</string>
   </property>
  </action>
  <action name="actionProfessorCad">
   <property name="text">
    <string>Professor</string>
   </property>
  </action>
  <action name="actionAlunoEmp">
   <property name="text">
    <string>Aluno</string>
   </property>
  </action>
  <action name="actionProfessorEmp">
   <property name="text">
    <string>Professor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
